src/__tests__/lib/containers/widgets/RadioGroup.test.tsx

```diff
@@ -38,70 +40,26 @@ describe('basic function', () => {
     expect(wrapper.find('.radio')).toHaveLength(props.options.length)
     expect(wrapper.find('input')).toHaveLength(props.options.length)
     expect(wrapper.find('div').at(0).hasClass('radioGroupClass')).toBe(true)
-    expect(
```

Comment:
Changes to existing tests in this file are whitespace changes caused by code formatter